src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -175,8 +175,6 @@ values of the `PeriodicExportingMetricReaderOptions`
 | `OTEL_METRIC_EXPORT_INTERVAL`                       | `ExportIntervalMilliseconds`                    |
 | `OTEL_METRIC_EXPORT_TIMEOUT`                        | `ExportTimeoutMilliseconds`                     |
 
-NOTE: `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE` is not supported yet [#3756](https://github.com/open-telemetry/opentelemetry-dotnet/issues/3756).
```

Comment:
Just mention with one liner that the environment variable is supported.

Reply:
please also call out the values suported. I dont think we support all the values. (like lowmemory)

Reply:
What is `lowmemory`?

Reply:
@cijothomas is support for `LowMemory` on the roadmap, or blocked by anything specific we can follow in the meantime?

- https://github.com/open-telemetry/opentelemetry-specification/blob/v1.24.0/specification/metrics/sdk_exporters/otlp.md#additional-configuration

Reply:
@utpilla The above link has the spec for `lowmemory`.
@austindrenski I don't think there are any blockers implementing lowmemory. Its just that nobody asked for it so far! Could you open an issue for it to better track adding it!